unit/src/RunnerTests.C

```diff
@@ -0,0 +1,39 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#include "Runner.h"
```

Comment:
Do we have a reference implementation of Runner? And build an input file for that as well?

Reply:
I do in some other branches, but those still have failing tests and issues that I'm trying to work out.  We wanted to get this up soon so that Yaqi can start building some of his stuff on it for some near term deadlines - while we work on reimplementing the fixedpoint/picard stuff in the new runner system in parallel.